Heapster in kubemark is crash-looping

Heapster in Kubemark is crashlooping with the following stacktrace:
```
I0822 12:20:09.975918       1 eventer.go:50] /eventer --source=kubernetes:https://104.197.233.84:443?inClusterConfig=0&useServiceAccount=0&auth=/kubeconfig/kubeconfig
I0822 12:20:09.976045       1 eventer.go:51] Eventer version 1.1.0
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x18 pc=0x81846b]

goroutine 1 [running]:
panic(0x1655320, 0xc820012060)
	/usr/lib/google-golang/src/runtime/panic.go:483 +0x3ef
k8s.io/heapster/common/kubernetes.GetKubeClientConfig(0xc8204e1a20, 0x100000002, 0x0, 0x0)
	/usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/common/kubernetes/configs.go:123 +0x3db
k8s.io/heapster/events/sources/kubernetes.NewKubernetesSource(0xc8204e1a20, 0xa, 0x0, 0x0)
	/usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/events/sources/kubernetes/kubernetes_source.go:179 +0x42
k8s.io/heapster/events/sources.(*SourceFactory).Build(0xc8204e1cc8, 0x7ffc50ee1def, 0xa, 0x7ffc50ee1dfa, 0x5, 0x0, 0x0, 0x0, 0x7ffc50ee1e02, 0x12, ...)
	/usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/events/sources/factory.go:33 +0xae
k8s.io/heapster/events/sources.(*SourceFactory).BuildAll(0xc8204e1cc8, 0xc82009dd40, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/events/sources/factory.go:46 +0x1ca
main.main()
	/usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/events/eventer.go:61 +0x4cd
```

@piosz @gmarek 
This seems to be configuration/test issue, but we should fix it.

--- a/test/kubemark/resources/heapster_template.json
+++ b/test/kubemark/resources/heapster_template.json
@@ -33,7 +33,7 @@
 				"containers": [
 				{
 					"name": "heapster",
-					"image": "gcr.io/google_containers/heapster:v1.1.0",
+					"image": "gcr.io/google_containers/heapster:v1.2.0-beta.1",
 					"resources": {
 						"requests": {
 							"cpu": "100m",
@@ -55,7 +55,7 @@
 				},
 				{
 					"name": "eventer",
-					"image": "gcr.io/google_containers/heapster:v1.1.0",
+					"image": "gcr.io/google_containers/heapster:v1.2.0-beta.1",
 					"resources": {
 						"requests": {
 							"cpu": "100m",

